site/src/sphinx/server-annotated-service.rst

```diff
@@ -0,0 +1,718 @@
+        @Get("/hello/{name}")
+        public HttpResponse hello(@Param("name") String name) {
+            return HttpResponse.of(HttpStatus.OK,
+                                   MediaType.PLAIN_TEXT_UTF_8,
+                                   "Hello, %s!", name);
+        }
+    });
+
+Mapping HTTP service methods
+----------------------------
+
+To map a Java method in an annotated HTTP service class to HTTP path, it has to be annotated with one of
+HTTP method annotations. Here is the list of HTTP method annotations that Armeria supports. Each of them
+is mapped to the HTTP method which has the same name.
+
+- `@Get`_
+- `@Head`_
+- `@Post`_
+- `@Put`_
+- `@Delete`_
+- `@Options`_
+- `@Patch`_
+- `@Trace`_
+
+To handle a single HTTP method in a single service method, you can annotate your service method simply,
+as follows.
+
+.. code-block:: java
+
+    class MyAnnotatedServiceClass {
+
+        @Get("/hello")
+        public HttpResponse hello() { ... }
+    }
+
+There are 5 PathMapping_ types provided for describing a path.
+
+- Exact mapping, e.g. ``/hello`` or ``exact:/hello``
+
+  - a service method will handle the path exactly matched with the specified path.
+
+- Mapping with path prefix, e.g. ``prefix:/hello``
+
+  - a service method will handle every path which starts with the specified prefix.
+
+- Mapping with path variables, e.g ``/hello/{name}`` or ``/hello/:name``
+
+  - a service method will handle the path matched with the specified path pattern. A path variable in the
+    specified pattern may be mapped to a parameter of the service method.
+
+- Mapping with regular expression, e.g. ``regex:^/hello/(?<name>.*)$``
+
+  - a service method will handle the path matched with the specified regular expression. If a named capturing
+    group exists in the regular expression, it may be mapped to a parameter of the service method.
+
+- Mapping with glob pattern, e.g. ``glob:/*/hello/**``
+
+  - a service method will handle the path matched with the specified glob pattern. A wildcard is mapped to
+    an index which starts with ``0``, so it may be mapped to a parameter of the service method.
+
+You can get the value of a path variable, a named capturing group of the regular expression or wildcards of
+the glob pattern in your service method by annotating a parameter with `@Param`_ as follows.
+Please refer to :ref:`parameter-injection` for more information about `@Param`_.
+
+.. code-block:: java
+
+    class MyAnnotatedServiceClass {
+
+        @Get("/hello/{name}")
+        public HttpResponse pathvar(@Param("name") String name) { ... }
+
+        @Get("regex:^/hello/(?<name>.*)$")
+        public HttpResponse regex(@Param("name") String name) { ... }
+
+        @Get("glob:/*/hello/**")
+        public HttpResponse glob(@Param("0") String prefix, @Param("1") String name) { ... }
+    }
+
+There are Java methods which are mapped to one HTTP method in the example above. What should you do if you want
```

Comment:
Every service method in the examples so far had a single HTTP method annotation with it. What if you want ..